sessions/git-and-github.qmd

```diff
@@ -1 +1,42 @@
-# Review of Git and GitHub {#sec-git-and-github}
\ No newline at end of file
+# Review of Git and GitHub {#sec-git-and-github}
+
+<!-- TODO: Add an introduction here -->
+
+## Learning objectives
+
+The **overall learning** outcome of this session is to:
+
+1.  Outline the core concepts and functionalities of Git and GitHub and
+    understand their role in facilitating effective collaboration
```

Comment:
Try to re-write this using Bloom's Taxonomy: https://www.simplypsychology.org/blooms-taxonomy.html

For instance "understand" is very vague. How will they know they understood?

Likewise, write it from the perspective of the learner. "outline" here isn't clear if we will be outlining it, or "you [the learner] will outline...".

Reply:
I'll change "understand" to something else. But in relatio to "outline", isn't it clear that it will be the learner that outlines due to L7? 

"This session’s **overall learning outcome** is to enable ***you*** to:

1.  Outline the core concepts ..."

Reply:
I think it's mainly an issue of language. How does one meaningfully "outline" concepts? You can outline tasks, or skills, or tools, but concepts? Is that all we want them to be able to do is say "Git tracks changes to files, GitHub has pull requests", etc. That doesn't say if they could describe or explain what those concepts are.

Reply:
Hmmmm... I guess I thought of it that way. Like [here](https://dictionary.cambridge.org/dictionary/english/outline), described in Cambridge Dictionary: 

![image](https://github.com/user-attachments/assets/21dc4147-f9df-4c13-b0cb-f37eb6babd0d)

Reply:
If you look under Blooms Taxonomy, it would fall under "remember" (as in memorize) section, not understanding. Since you used the word understanding earlier, I assumed you wanted more than listing recalled facts about something. But maybe that's what you want them to do at the end of this session? What is it you want them to do at the end? Could you describe that in more detail so we can agree on the right term for it?

Reply:
But we could for sure use another word than "outline". What do you suggest?